common/styles/scss/_switch-user-accounts.scss

```diff
@@ -0,0 +1,12 @@
+#switchUserAccounts {
```

Comment:
Since by default `app.scss` is disabling the `overflow-y` (and therefore disabling scroll), you should add a rule to override this. That's how I did it in mdHelp page before:

https://github.com/informatics-isi-edu/chaise/blob/master/recordedit/mdHelpStyle.min.css#L6